packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cli.integtest.ts

```diff
@@ -1,6 +1,7 @@
 import { promises as fs, existsSync } from 'fs';
 import * as os from 'os';
 import * as path from 'path';
+import * as CFN from '@aws-sdk/client-cloudformation';
```

Comment:
I noticed that `fixture` has its own CFN client, so I should use that instead of this, probably.